src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -12,18 +10,37 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class OptionsServiceCollectionExtensions
     {
-        public static IServiceCollection AddOptions(this IServiceCollection services)
+        public static IServiceCollection AddOptionsMonitor<TOptions>(this IServiceCollection services) where TOptions : class, new()
         {
             if (services == null)
             {
                 throw new ArgumentNullException(nameof(services));
             }
 
-            services.TryAdd(ServiceDescriptor.Singleton(typeof(IOptions<>), typeof(OptionsManager<>)));
             services.TryAdd(ServiceDescriptor.Singleton(typeof(IOptionsMonitor<>), typeof(OptionsMonitor<>)));
             return services;
         }
 
+
+        public static IServiceCollection AddOptions<TOptions>(this IServiceCollection services) where TOptions : class, new()
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddSingleton(serviceProvider =>
```

Comment:
None of our repos are going to call the `AddOptions<T>` sugar by default, instead they will register the singleton in our AddXyz methods.  Apps or other frameworks are free to use the sugar method to avoid requiring complicated ctors for their options.  

Services can certainly adopt the 2 ctors pattern as well.  But I don't think we need to do that in general since most of our frameworks fall over anyway if AddXyz isn't called.